src/new_index/mempool.rs

```diff
@@ -285,6 +285,48 @@ impl Mempool {
         &self.backlog_stats.0
     }
 
+    pub fn old_txids(&self) -> HashSet<Txid> {
+        return HashSet::from_iter(self.txstore.keys().cloned());
+    }
+
+    pub fn download_new_mempool_txs(daemon: &Daemon, old_txids: &HashSet<Txid>, new_txids: &HashSet<Txid>) -> Vec<Transaction> {
```

Comment:
I usually prefer to place static methods at the end of the `impl` block, separately from the instance methods